release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2129,6 +2129,9 @@ With this update, the Cluster Network Operator `allowlist` controller monitors i
 
 * Previously, for Intel E810 NICs, resetting a MAC address on an SR-IOV with a virtual function (VF) when a pod was deleted caused a failure. This resulted in a long delay when creating a pod with SR-IOV VF. With this update, the container network interface (CNI) does not fail fixing this issue. (link:https://issues.redhat.com/browse/OCPBUGS-5892[*OCPBUGS-5892*])
 
+* Previously, an issue was observed in the {product-title} with some pods getting stuck in the `terminating` state. This affected the reconciliation loop of the allowlist controller resulting in unwanted retries which caused the creation of multiple pods.
```

Comment:
Suggestion only, but felt like the 2nd sentence needed a break or two or somehow. Maybe something like:
```suggestion
* Previously, an issue was observed in the {product-title} with some pods getting stuck in the `terminating` state. This affected the reconciliation loop of the allowlist controller, which resulted in unwanted retries and caused the creation of multiple pods.
```
Or:
```suggestion
* Previously, an issue was observed in the {product-title} with some pods getting stuck in the `terminating` state. This affected the reconciliation loop of the allowlist controller, which resulted in unwanted retries that caused the creation of multiple pods.
```